server-side/profileMngr.js

```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+export default function activeProfile(userId) {
+    const hourNow = new Date().toLocaleTimeString('en-US', { hour12: false }).split(':')[0];

```

Comment:
time zone should be taken from user's settings, ask Moria Meshulami about this data
also rename to currentTime, or currTime

Reply:
I fixed it